### PR TITLE
Refactored borrowbook

### PR DESCRIPTION
**Pull request by aatish-rajkarnikar**
Refactored borrowbook

### CODE REVIEW

File: src/library/borrowbook/BorrowBookControl.java [hunk 1]
```diff
@@ -9,116 +9,116 @@
...
 public class BorrowBookControl {
 	
-	private BorrowBookUI uI;
+	private BorrowBookUI ui;
 	
-	private Library lIbRaRy;
-	private Member mEmBeR;
+	private Library library;
+	private Member member;
 	private enum CONTROL_STATE { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
```
Comment (parneetbatra) [inline src/library/borrowbook/BorrowBookControl.java]: As per code guide like enum must start with an uppercase letter. CONTROL_STATE must ControlState. Please have a look in code guideline document. Thanks.

Comment (aatish-rajkarnikar) [reply src/library/borrowbook/BorrowBookControl.java]: @parneetbatra  i think it's been fixed already. can you verify?

File: src/library/borrowbook/BorrowBookUI.java [hunk 2]
```diff
@@ -58,19 +58,19 @@ public void RuN() {
...
 			case RESTRICTED:
-				iNpUT("Press <any key> to cancel");
+				input("Press <any key> to cancel");
 				control.cancel();
 				break;
 			
 				
 			case SCANNING:
-				String BoOk_StRiNg_InPuT = iNpUT("Scan Book (<enter> completes): ");
+				String BoOk_StRiNg_InPuT = input("Scan Book (<enter> completes): ");
```
Comment (parneetbatra) [inline src/library/borrowbook/BorrowBookUI.java]: Please BoOk_StRiNg_InPuT change to bookStringInput.

Comment (aatish-rajkarnikar) [reply src/library/borrowbook/BorrowBookUI.java]: Fixed!

**Comment by ttawan** [comment]
All good

**Review by parneetbatra — APPROVED** [review]

**Pull request merged**